jareff-g/ALT-Scann8
# ALT-Scann8 1.9.0 (#68)

* ALT-Scann8 1.8.40 (#59)

- HDR merge-in-place now works. But, as anticipated, is too slow (around 11 FPM, vs 36 FPM without). Rough extrapolation for RPi5 (x2.5 RPi4) would be 27 FPM, faster but still slow

* ALT-Scann8 1.8.41 (#61)

- Review of alignment of widgets, make them more evenly distributed, also in small screen mode

* ALT-Scann8 1.8.42 (#62)

- Fix some remaining alignment issues
- Display popup warning when running out of disk space
- Add disk space check to simulated capture loop (to test new popup)

* ALT-Scann8 1.8.43 (#63)

- Errors when starting in experimental or expert mode alone (only working when using both together)

* ALT-Scann8 1.8.43 (#64)

Arduino 1.0.3
- Bugfix for slow forward auto-stop. UV Led was switched off, so film detection could not work

* Plotter threshold (#65)

* ALT-Scann8 1.8.43 Arduino 1.0.3
- Add line with threshold level to integrated plotter
- Bugfix: Disable 'Merge in place' checkbox when HDR disabled

* ALT-Scann8 1.8.43 Arduino 1.0.3
- Add line with threshold level to integrated plotter
- Bugfix: Disable 'Merge in place' checkbox when HDR disabled

* ALT-Scann8 1.8.45
- Bugfix: Due to a problem with data arriving from Arduino via I2C, sometimes the threshold level was wrong. Root cause is unknow, for now we prevent plotting the wrong value

* Updated tooltip.py to latest version

* ALT-Scann8 1.8.46 (#66)

Arduino 1.0.5
- Extra steps can now be set to negative values (-30 to +30). When negative, they are subtracted from the minimum number of steps where Arduino will start considering the photo-transistor, if positive is still the same, unconditional number os steps are done after frame is detected, before snapshot is taken. Normally negative values shoudl not be needed, but today I had a weird case where even after physically adjusting the film gate position, frames were being detected too high. Need to check the scanner to see if there is something wrong. But then, it was the second movie of the day, and the first went perfectly fine (it had the same characteristics as the one with the problem)

* Multi resolution capture (#67)

* ALT-Scann8 1.8.47
- Replace file type (jpg/png) radio buttons with drop down
- Replace 'HQ' button with drop down
- Additional resolutions added in drop-down (although minimum captor in HQ camera is "2028x1520" at the minimal, so no speed gain (need to search how to use format SRGGB10_CSI2P)
- Remove VideoCaptureMode as it served no purpose
- Move Multi-exposure frame to experimental area (as final results are not good) -

* ALT-Scann8 1.8.48
- Include raw mode 1332x990 (120.05 fps) in experimental mode (-x) only. This allows faster scanning (350FPM with no stabilization delay), but requires to physically open the zoom in the lens since it appear sto use only the center of the sensor. Not very useful, but included anyhow
- Log camera sensor modes omn startup, for info

* ALT-Scann8 1.8.48
- Fix 3 pyflakes minor issues

* ALT-Scann8 1.8.48
- Save window position upon exit to restore it in the next run

* ALT-Scann8 1.8.48
- Reduce font of rewind/FF buttons (when clicking of them were automatically getting bigger to accommodate 'Stop' text)
- Remove a lot of 'Win.update()' that were not really needed (they were causing scan process to become progressive slower)

* ALT-Scann8 1.9.0
- Set version to 1.9.0 before merging to develop, and then to master
